cmd/kops/create_cluster.go

```diff
@@ -369,6 +370,10 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 	}
 	cluster.Spec.Channel = c.Channel
 
+	if featureflag.EnableExternalCloudController.Enabled() {
```

Comment:
Not sure if this is needed or not, but I added it anyways since this is the only way to enable CCM now.

Reply:
We don't even need it here.  This is simply to get the API in unwired.